test/chemistry/test_oovqe.py

```diff
@@ -0,0 +1,121 @@
+from qiskit import Aer
+
+
+@ddt
+class TestOOVQE(QiskitAquaTestCase):
+    """ Test of the OOVQE algorithm"""
+
+    def setUp(self):
+        super().setUp()
+        self.energy_vqe = -2.77  # energy of the VQE with pUCCD ansatz and LBFGSB optimizer
+        self.initial_point = [0.039374, -0.47225463, -0.61891996, 0.02598386, 0.79045546,
+                              -0.04134567, 0.04944946, -0.02971617, -0.00374005, 0.77542149]
+        self.seed = 50
+        aqua_globals.random_seed = self.seed
+        driver = PySCFDriver(
+            atom="H 1.738000 .0 .0; H .15148 1.73139 .0; H -1.738 .0 .0; H -0.15148 -1.73139 .0",
+            unit=UnitsType.ANGSTROM,
+            charge=0,
+            spin=0,
+            basis='sto3g')
+        self.qmolecule = driver.run()
+        self.core = Hamiltonian(transformation=TransformationType.FULL,
+                                qubit_mapping=QubitMappingType.PARITY,
+                                two_qubit_reduction=False,
+                                freeze_core=False,
+                                orbital_reduction=[])
+
+        algo_input = self.core.run(self.qmolecule)
+        self.qubit_op = algo_input[0]
+
+        init_state = HartreeFock(
+            num_orbitals=self.core._molecule_info['num_orbitals'],
+            qubit_mapping=self.core._qubit_mapping,
+            two_qubit_reduction=self.core._two_qubit_reduction,
+            num_particles=self.core._molecule_info['num_particles'])
+
+        self.var_form = UCCSD(
+            num_orbitals=self.core._molecule_info['num_orbitals'],
+            num_particles=self.core._molecule_info['num_particles'],
+            active_occupied=None, active_unoccupied=None,
+            initial_state=init_state,
+            qubit_mapping=self.core._qubit_mapping,
+            two_qubit_reduction=self.core._two_qubit_reduction,
+            num_time_slices=1,
+            method_doubles='pucc',
+            same_spin_doubles=False,
+            method_singles='both',
+            skip_commute_test=True,
+            excitation_type='d')
+
+        self.quantum_instance = QuantumInstance(Aer.get_backend('statevector_simulator'),
+                                                shots=1,
+                                                seed_simulator=self.seed,
+                                                seed_transpiler=self.seed)
+        self.optimizer = COBYLA(maxiter=1)
+        self.algo = OOVQE(operator=self.qubit_op,
+                          var_form=self.var_form,
+                          optimizer=self.optimizer,
+                          core=self.core,
+                          qmolecule=self.qmolecule,
+                          expectation=MatrixExpectation(),
+                          initial_point=self.initial_point)
+
+    def test_orbital_rotations(self):
+        """Test that orbital rotations are performed correctly."""
+
+        self.algo.optimizer.maxiter = 1
```

Comment:
Is this low iter number the reason for the "special" initial point - I would infer from this that the point is the solution?

Reply:
Yes, it just to test that at the rotation is performed correctly at that point (which is very close to converged solution of OOVQE for that system).